README.md

```diff
@@ -0,0 +1 @@
+Hello world
```

Comment:
- [ ] Still, this issue is not resolved. If you are struggling, you can use [microverse template](https://github.com/microverseinc/readme-template) to make a descriptive and professional README. :slightly_smiling_face: